itl/plugins-contrib.d/network-components.conf

```diff
@@ -833,3 +833,266 @@ object CheckCommand "printer_health" {
 	vars.printer_health_hostname = "$check_address$"
 	vars.printer_health_mode = "supplies-status"
 }
+
+template CheckCommand "generic-thola-check-command" {
+    import "plugin-check-command"
+
+    command = [ PluginContribDir + "/thola-client", "check" ]
+
+    arguments += {
+        "--target-api" = {
+            required = true
+            value = "$thola_api_address$"
+            description = "Address of the thola API"
+        }
+    }
+}
+
+template CheckCommand "generic-thola-device-check-command" {
+    import "generic-thola-check-command"
+
+    arguments += {
+        "(no key.1)" = {
+            order = 0
+            required = true
+            skip_key = true
+            value = "$thola_device_address$"
+            description = "Thola Check Device Mode"
+        }
+        "--snmp-community" = {
+            required = false
+            value = "$thola_device_snmp_community$"
+            description = "SNMP Community of target"
+        }
+        "--snmp-version" = {
+            required = false
+            value = "$thola_device_snmp_protocol$"
+            description = "SNMP Version of target"
+        }
+    }
+
+    vars.thola_device_address = "$check_address$"
```

Comment:
`$check_address$` comes from [here](https://github.com/Icinga/icinga2/blob/8509e55b78a82ee1f11621a1884ee056852ab179/itl/command-plugins.conf#L3-L17), but you didn’t import the template, did you?

Reply:
> We made a mistake. We chose the wrong variable name, we wanted to set it to the host ip address, this should be $address$ then, right?

No, $check_address$ handles IPv4/6, I'd prefer that.

Reply:
Yeah, I already changed it back again. But the commit doesn't show up in the pull request although I pushed it.